app/src/main/java/org/simple/clinic/patient/PatientSearchResult.kt

```diff
@@ -144,6 +144,27 @@ data class PatientSearchResult(
         ORDER BY P.fullName COLLATE NOCASE ASC LIMIT :limit
     """)
     fun searchByPhoneNumber(phoneNumber: String, limit: Int): List<PatientSearchResult>
+
+    @Suppress("AndroidUnresolvedRoomSqlReference")
+    @Query("""
+        SELECT * FROM 
+        PatientSearchResult searchResult
+        LEFT JOIN Patient P ON P.uuid = searchResult.uuid
+        WHERE phoneNumber LIKE '%' || :phoneNumber || '%' AND P.deletedAt IS NULL
+        ORDER BY P.fullName COLLATE NOCASE ASC 
+    """)
+    fun searchByPhoneNumberNew(phoneNumber: String): DataSource.Factory<Int, PatientSearchResult>
```

Comment:
Can we rename this to something like `searchByPhoneNumberPaginated`? Same for `searchByName` since we are going to have both the flows present for some time in the app.